lib/mvi/responses/profile_parser.rb

```diff
@@ -135,6 +135,12 @@ def get_patient_name(patient)
         locate_element(patient, NAME_XPATH)
       end
 
+      def sanitize_edipi(edipi)
+        return nil unless edipi
```

Comment:
Should we change this to, `return if edipi.nil?`? I think it reads better. If no argument is passed it will be an ArgumentError, we really need to just handle the argument being nil, so lets make that more clear.